backend/tracim_backend/error.py

```diff
@@ -77,6 +77,10 @@ class ErrorCode(IntEnum):
     USER_CANT_DELETE_HIMSELF = 3004
     USER_CANT_REMOVE_IS_OWN_ROLE_IN_WORKSPACE = 3005
     USER_CANT_CHANGE_IS_OWN_PROFILE = 3006
+    # TODO - G.M - 2019-10-10 - remove WORKSPACE_LABEL_ALREADY_USED error
+    # 3007 error code is now deprecated. Let this code
+    # here now to avoid reusing it soon (frontend may should use it and create a
+    # another exception with same value may cause weird result)
```

Comment:
As you delete `WorkspaceLabelAlreadyUsed` backend will not use  ̀WORKSPACE_LABEL_ALREADY_USED` anymore right ? So Frontend will not see this error code. So we can delete this constant i think.